piglow/examples/strobe.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"time"
+
+	"github.com/goiot/devices/piglow"
+	"golang.org/x/exp/io/i2c"
+)
+
+func main() {
+	p, err := piglow.Open(&i2c.Devfs{Dev: "/dev/i2c-1", Addr: 0x54})
+	if err != nil {
+		panic(err)
+	}
+
+	defer p.Close()
+
+	if err := p.Reset(); err != nil {
+		panic(err)
+	}
+
+	for {
```

Comment:
Currently the `Close` function will not reset the internal registers of the piglow. If you want to clear it out you have to call `Reset`. There is also a `Shutdown` command, which I did not implement, that will clear out the registers and "turn off" the SN3218.